### ISSUE TITLE
Bug in flow type

### ISSUE THREAD

**Issue by steida** [jlongster/prettier#476]
Before

```js
type TextInputContext = {
  TextInput: () => React.Element<*>,
  theme: Theme,
};
```

After

```js
type TextInputContext = { TextInput(): React.Element<*>, theme: Theme };
```

`TextInput()` is wrong

**Comment by apalm** [comment]
@steida See https://github.com/jlongster/prettier/issues/310.

**Comment by vjeux** [comment]
Thanks @steida

The two forms are equivalent, so it is technically working. But indeed we should use the other one. This is the last bug on my plate!

**Comment by vjeux** [comment]
This is now fixed in 0.16.0 fyi ;)
